frontend/public/components/utils/dropdown.jsx

```diff
@@ -306,7 +306,10 @@ export class Dropdown extends DropdownMixin {
     const rows = [];
     const bookMarkRows = [];
 
-    _.each(items, (content, key) => {
+    const sortedKeys = this.props.sortItemKeys && this.props.sortItemKeys() || _.keys(items);
```

Comment:
Any reason this needs to be a function vs just an array of keys?

Reply:
Just that sort APIs tend to be callback function based, so was thinking it might provide consistency/flexibility for future uses if we ever need it.  

I'm fine with changing it to just an array of keys if you think that is preferable.

Reply:
A function here might actually cause problems. The child component won't rerender if the `sortItemKeys` function doesn't change, even if the key ordering has changed.

Let's pass an array.

Reply:
Which child component are you referring to?

Thinking this through. If the prop is called `sortItemKeys`, a consumer should assume its tied to `items`.  Changes to `items` and the result of `sortItemKeys` should be in sync, or it should at least be safe to assume they will be in sync & render output should be reliable. 

Or perhaps not?  As a `callback`, I guess it ensures the `sort` runs whenever anything triggers `render`.  There is def an optimization question worth asking, at least. 

Worth the discussion, but I'll update it to an array for this one.